component/portal/src/main/java/org/exoplatform/portal/mop/user/UserNode.java

```diff
@@ -168,11 +176,21 @@ public String getResolvedLabel() {
     String id = context.getId();
 
     //
+    Locale userLocale = owner.navigation.portal.context.getUserLocale();
     if (context.getState().getLabel() != null) {
       ResourceBundle bundle = owner.navigation.getBundle();
-      resolvedLabel = ExpressionUtil.getExpressionValue(bundle, context.getState().getLabel());
+      if (bundle == EmptyResourceBundle.INSTANCE) {
+        ResourceBundleService resourceBundleService = ExoContainerContext.getService(ResourceBundleService.class);
```

Comment:
We can not refer social in gatein-portal:

- You can just base on sharedResourceBundleNames in all cases and add the missed **SpaceActivityStreamPortlet.label.name** label in https://github.com/Meeds-io/social/blob/develop/component/service/src/main/resources/locale/social/menu/webui_en.properties
- No other social conditions needed in that case